vortex-array/src/lib.rs

```diff
@@ -52,7 +52,7 @@ pub mod iter;
 mod metadata;
 pub mod stats;
 pub mod stream;
-mod tree;
+pub mod tree;
```

Comment:
this is to make TreeDisplay be visible in Docs. Right now there is a tree_display method but it doesn't have a link to navigate to its source